### PR TITLE
[homework #2] - HTML by Nicolas Roberto

### PR DESCRIPTION
**Pull request by nicolasroberto**
Names fixed

### CODE REVIEW

**Comment by mmelnyk-ua** [comment]
@nicolasroberto 
Please improve the next things:
1.  I can't download your resume (I have 403 error)
2. Add validation for the contact form (please take a look at the task description which fields should be required).
3. For phone number you are using `type='number'` but we can use `type=tel`: https://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_tel

File: homeworks/nicolas-roberto/nr-02-html/contact.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          type="text"
+          id="lastName"
+          placeholder="Your last name.."
+          required
+        />
+
+        <br />
+
+        <label for="phone"><strong>Phone number*</strong></label>
+        <input type="phone" id="phone" placeholder="Your phone.." required />
```
[the remark this answers is not in the document]
Comment (nicolasroberto) [reply homeworks/nicolas-roberto/nr-02-html/contact.html:34]: Sorry, my bad.

[the remark this answers is not in the document]
Comment (solvd-mmelnyk) [reply homeworks/nicolas-roberto/nr-02-html/contact.html]: @nicolasroberto thanks for fixing!

**Comment by solvd-mmelnyk** [comment]
Looks good to me! I will merge this PR after tomorrow's lecture to show all students examples!

File: homeworks/nicolas-roberto/nr-02-html/contact.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicolasroberto) [reply homeworks/nicolas-roberto/nr-02-html/contact.html]: No problem, thanks for all the suggestions!

**Review by solvd-mmelnyk — APPROVED** [review]

**Pull request merged**